docs/en_US/Compression/AutoCompression.rst

```diff
@@ -0,0 +1,101 @@
+Auto Compression with NNI Experiment
+====================================
+
+This approach is mainly a combination of compression and nni experiments.
+It allows users to define compressor search space, including types, parameters, etc.
+Its using experience is similar to launch the NNI experiment from python.
```

Comment:
Just personal concern, maybe not correct. I think the doc mainly focuses on how to use this new feature, but it doesn't tell users what things this feature can actually help them do clearly and users will be confused or misunderstand. 
In my opinion, this feature can help users try different compression algorithms including pruning algorithms and quantization algorithms by adding them into our 'search space'. By using it, users can easily choose different compression algorithms and apply them to model to get feedback easily and automatically. But If I am a branch new user, after reading this doc, I can't get this key point and still miss some important information such as 
- what 'search space', 'types' and parameters mean?
- what is the meaning of 'auto compress'?
- what is the meaning of 'combination of compression and nni experiments'?
- If I want to try different compression algorithms, will they be applied together(apply pruning algorithm and quantization to model simultaneously) or single sequentially?

Reply:
Good suggestion, will add more descriptions and explanations about auto compress and what this can help.

Reply:
Rewrite the doc and welcome more comments.